docs/_posts/0000-05-02-oo-polymorphism.md

```diff
@@ -1,12 +1,8 @@
 ## Polymorphism
 
 + What is it?
-+ How do we use it?
-    + Mainly via inheritance mechanisms
-    + Also available (where required) via interfaces
-+ Why is it useful?
-    + Simplifies our code
-    + Makes our code more re-usable
```

Comment:
~~Good that you’ve introduced a description. Is there any reason to remove the other bits?~~

I see this has moved now.